src/Microsoft.XmlSerializer.Generator/tests/System.Xml.XmlSerializer.Sgen.Data.XmlSerializers.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <ProjectGuid>{1BCAF5A7-574E-4E74-9147-0572344C2C93}</ProjectGuid>
+    <NoWarn>CS0219</NoWarn>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'">
+    <!--<CodeAnalysisRuleSet>.\CodeAnalysis.ruleset</CodeAnalysisRuleSet>-->
+    <!--<OutputPath>..\..\..\bin\AnyOS.AnyCPU.Debug\System.Xml.XmlSerializer.Sgen.Tests\netstandard\</OutputPath>-->
+    <!--<TreatWarningsAsErrors>false</TreatWarningsAsErrors>-->
+    <OutputPath>..\..\..\bin\AnyOS.AnyCPU.Debug\System.Xml.XmlSerializer.Sgen.Data.XmlSerializers\netstandard\</OutputPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
+  <PropertyGroup>
+    <AssemblyName>System.Xml.XmlSerializer.Sgen.Data.XmlSerializers</AssemblyName>
```

Comment:
Is this the baseline? Will it be clearer if we call it `Microsoft.XmlSerializer.Generator.Baseline.Tests`?